assets/js/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 class awesomBooks {
```

Comment:
- [ ] Please enable all the linter checks in all your files, you can leave the one for `Luxon.js` library. Linters are required to ensure that your code is of the Microverse standard and free from errors. Kindly enable all linters and fix the linter errors.

Reply:
Thank you @julie-ify for nice comment and corrections. I have enabled all linters. But still I need to disable some of that , such as undefined function, and JS extension. Because the functions are defined in one file and used in another one.